ext/File-Glob/t/basic.t

```diff
@@ -3,13 +3,12 @@
 BEGIN {
     chdir 't' if -d 't';
     @INC = '../lib';
-    require Config; Config->import;
+    use Config;
```

Comment:
this is incorrect and not honoring the `@INC` setup

Reply:
why was this change even made? It does not seem to be needed for adding strict+warnings.  Only necessary changes should be made in these commits.

Reply:
when enabling `strict, warnings & co` as performed on this experimental branch, it complains about `%Config` not being declared. The exact error message is:
`Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?)`

Reply:
Perhaps it would be better to change the following `$Config{'extensions'}` to `$Config::Config{'extensions'}` to avoid the issue.

Reply:
This is one of the two solutions to fix the issue. I was using that `Config::Config` trick earlier until I found a better alternate, by adding just before the `BEGIN` block `our %Config`.... less changes, all the most when `%Config` is used multiple times